keyboards/anavi/macropad8/keymaps/default/keymap.c

```diff
@@ -0,0 +1,68 @@
+#include QMK_KEYBOARD_H
+
+#define _MAIN 0
+#define _FN 1
```

Comment:
In general, we prefer an enum here, rather than defines.  It makes things easier to manage, long term. 

However, it's not strictly necessary here, and this is more of a nitpick.